cmd/ftl/cmd_call.go

```diff
@@ -39,8 +41,43 @@ func (c *callCmd) Run(ctx context.Context, client ftlv1connect.VerbServiceClient
 	if err != nil {
 		return fmt.Errorf("invalid request: %w", err)
 	}
+
+	logger.Debugf("Calling %s", c.Verb)
+	// lookup the verbs
+	res, err := ctlCli.GetVerbs(ctx, connect.NewRequest(&ftlv1.VerbsRequest{}))
```

Comment:
We'd only want to do the distance check if we _don't_ find a verb, rather than on every call, as it's expensive in time and compute. You can do this by checking if the returned connect error is a `connect.CodeNotFound` with something like this:

```go
if cerr := new(connect.Error); errors.As(err, &cerr) && cerr.Code() == connect.CodeNotFound {
	// ...
}
```

The server should be returning `CodeNotFound` correctly, but you might need to confirm.

Reply:
you mean:
1. make the "call" regardless,
2. if CodeNotFound, fetch the schema
3. build up suggestions

did I get that right?

Reply:
Yes, exactly right.

Reply:
`if cerr := new(connect.Error); errors.As(err, &cerr) && cerr.Code() == connect.CodeNotFound {`

doesn't work. the error has no code. apart from that, it's good to go. I'll figure out how to make it work

Reply:
fixed it, but I had to introduce the connect error in that branch in the controller. and also an `ErrNotFound` in the schema. take a look.